homeassistant/components/bond/config_flow.py

```diff
@@ -42,32 +47,73 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
-    async def async_step_user(self, user_input=None):
-        """Handle the initial step."""
+    _discovered: dict = None
+
+    async def async_step_zeroconf(
+        self, discovery_info: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, Any]:
+        """Handle a flow initialized by zeroconf discovery."""
+        name: str = discovery_info["name"]
+        host: str = discovery_info["host"]
```

Comment:
you can use CONF_NAME and CONF_HOST as the keys here

Reply:
got it.